tests/unit/nupic/research/spatial_pooler_unit_test.py

```diff
@@ -1315,7 +1315,7 @@ def testInhibitColumnsLocal(self):
                         #   L  W  W  L  L  W  W   L   L    L
     trueActive = [1, 2, 5, 6]
     active = list(sp._inhibitColumnsLocal(overlaps, density))
-    # self.assertListEqual(trueActive, active)
+    # self.assertListEqual(trueActive, active) #FIXME why was this commented out? should still be used?
```

Comment:
Someone? It does not work right off, and I don't intend to fix it in this PR, but someone cloud know or we should make an issue for it, or remove the part of the test.